### PR TITLE
Use white Layer5 logo for dark mode

### LINKED ISSUE #3259: Create a Layer5 logo with white text
**Current Behavior**
In dark mode, the current logo is barely legible, as a result of the dark text color.

dark-text-logo

**Desired Situation**
Create a new logo with white "LAYER" text by editing the one found in this repo (src/assets/images/layer5/layer5-only/svg/layer5-no-trim.svg). This new logo will be used to fix the readability in dark mode.

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by NPDebs**
Signed-off-by: Deborah Udoh <deborahudoh02@gmail.com>

 **Description**
This PR fixes #3259. 
Screenshot_20221004-134028_Chrome

**Notes for Reviewers**
In dark mode, this logo with white text replaces the former.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 423379c8c3df6453afe9995bcfb2abaaf58d9fac at: https://633c301f20b466104a12a76d--layer5.netlify.app

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 <p style="text-align:center;" align="center">
-  <img align="center" src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/layer5-no-trim.svg" alt="Layer5-logo" width="45%" />
+   <picture align="center">
```
Comment (legendarykamal) [inline README.md:2]: Will need improvement, dark mode is ok but light mode is buggy 
image

**Review by legendarykamal — CHANGES_REQUESTED** [review]
Make sure to maintain the same aspect ratio in both light and dark mode.

File: README.md (hunk 1 above)
Comment (NPDebs) [reply README.md:2]: That's weird. It looks un-buggy over here. 🤔

**Comment by NPDebs** [comment]
Sure @legendarykamal . Thank you.

File: README.md (hunk 1 above)
Comment (NPDebs) [reply README.md:2]: Ah...yes.
I found the perfect image to normalise aspect ratio!

Commiting changes in 2 seconds. 🙂🚀

**Comment by l5io** [comment]
🚀 Preview for commit 20e46b785619176fed2f52f657366530888143b3 at: https://633d5048808d35692b27706a--layer5.netlify.app

**Review by legendarykamal — APPROVED** [review]
Looks so fabulous in dark mode. Thankyou @NPDebs 💯  

image

**Comment by NPDebs** [comment]
Well, that's very reassuring. 😃😊 Thanks @legendarykamal 

Couldn't have done it without you! 🙌🏼

File: README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+      <source media="(prefers-color-scheme: dark)" srcset="./src/assets/images/layer5/layer5-only/svg/layer5-light-no-trim.svg">
```
Comment (Nikhil-Ladha) [inline README.md:3]: Preferably, make a copy of this image under the `.github/assets/images/layer5`, so that there is no interdependency.

Comment (NPDebs) [reply README.md:3]: Sure. On it!

**Comment by NPDebs** [comment]
Changes made @Nikhil-Ladha  🤝🏻 Thank you.

**Comment by l5io** [comment]
🚀 Preview for commit 71c995a28e431813f1577ec50157671d5d18b77c at: https://633dc5eab76e7907598a7cec--layer5.netlify.app

**Pull request merged**